src/docs/asciidoc/resources/migration_guide_to_3.13.adoc

```diff
@@ -12,4 +12,8 @@
 When calling an opfab API you cannot use anymore trailing slash on url.
 
 
-If, for example, one makes a request like https://opfab/businessconfig/processes/, it must be replace by https://opfab/businessconfig/processes
\ No newline at end of file
+If, for example, one makes a request like https://opfab/businessconfig/processes/, it must be replace by https://opfab/businessconfig/processes
+
+== viewCardInAgenda Parameter renamed
+
+The viewCardInAgenda parameter has been renamed viewCardInCalendar to synchronize the names of the different instances. 
```

Comment:
The viewCardInAgenda (parameter in object return by usercard template in  getSpecificCardInformation)  has been renamed viewCardInCalendar to be consistant with other part of the software . So you may have to update your usercard templates.